src/cbc_sdk/platform/devices.py

```diff
@@ -370,9 +405,6 @@ def _query_implementation(cls, cb, **kwargs):
         Args:
             cb (BaseAPI): Reference to API object used to communicate with the server.
             **kwargs (dict): Not used, retained for compatibility.
-
```

Comment:
Should we have the Raises section here or it is useless considering it was stated that it is not used?

Reply:
From the Docstring Style Guide: "Exceptions that get raised if the API in the docstring is violated **SHOULD NOT** be documented here, because this would mean that behavior under violation of the API would be _part_ of the API."